### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_NotebookQueryTable_ScreenReader:Voice over is announcing the column header two times for the table.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5683]
Check out Keros! - Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS4.2.3; #MAS1.3.1; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio
Version: 1.8.0-insider (user setup)
Commit: 2ca39b5
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Tools Name: Voice Over
MAS Number :MAS1.3.1,MAS4.2.3

**Steps to Reproduce :**
1.Install the app Azure Data Studio.
2.Connect to the Database->Go to file->New Notebook.
3.Click on “Code” button and write query in the box for example SELECT * FROM sys.traces;
4.Select query->Click on the Run command.
5.Start Voice Over.
6.Navigate to table.
7.Observe that Screen reader is announcing the column header twice for the table.

**Actual:**
Screen reader is announcing the column header twice for the table.

**Expected:**
Screen reader should announce the column header once for the table.

**User Impact:**
User with disabilities will not be able to understand why the screen reader reading that column header twice for the table.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**

**MAS4.2.3**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={30e4fb40-dcc1-45b4-b714-5e2c28b23ae8}

**MAS1.3.1**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc=%7B54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4%7D&cid=1a1d7dfd-3dfd-4394-a8af-81fa7c1cf482

**Attachment for Reference:**
[5683_[Accessibility]A11y_AzureDataStudio_NotebookQueryTable_ScreenReader.pptx](https://github.com/microsoft/azuredatastudio/files/3227321/5683_.Accessibility.A11y_AzureDataStudio_NotebookQueryTable_ScreenReader.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by chlafreniere** [comment]
Note: this appears to also affect query editor. 

![image](https://user-images.githubusercontent.com/40371649/60225746-f78a3380-983c-11e9-945b-ed6b1436c261.png)

**Comment by udeeshagautam** [comment]
Known issue same as: https://github.com/microsoft/azuredatastudio/issues/6909
This can be close since this happen one when moving away or towards as header

**Comment by udeeshagautam** [comment]
Confirmed with accessibility SME Alexandria Reynolds and closing.
